lxd-agent/events.go

```diff
@@ -141,15 +140,78 @@ func eventsProcess(event api.Event) {
 		return
 	}
 
-	// Attempt to perform the mount.
-	mntSource := fmt.Sprintf("lxd_%s", e.Name)
+	if e.Action == "added" {
+		// Attempt to perform the mount.
+		mntSource := fmt.Sprintf("lxd_%s", e.Name)
 
-	_ = os.MkdirAll(e.Config["path"], 0755)
-	_, err = shared.RunCommand("mount", "-t", "virtiofs", mntSource, e.Config["path"])
-	if err != nil {
-		logger.Infof("Failed to mount hotplug %q (Type: %q) to %q", mntSource, "virtiofs", e.Config["path"])
+		_ = os.MkdirAll(e.Config["path"], 0755)
+		_, err = shared.RunCommand("mount", "-t", "virtiofs", mntSource, e.Config["path"])
+		if err != nil {
+			logger.Infof("Failed to mount hotplug %q (Type: %q) to %q", mntSource, "virtiofs", e.Config["path"])
+			return
+		}
+
+		logger.Infof("Mounted hotplug %q (Type: %q) to %q", mntSource, "virtiofs", e.Config["path"])
+	} else if e.Action == "removed" {
+		targetPath := e.Config["path"]
+
+		if !filepath.IsAbs(targetPath) {
+			logger.Infof("the device path must be absolute: %q", e.Config["path"])
```

Comment:
Shall we make this a logger.Warnf?

Also why lower case `t` at start of sentence?